module-4/hashcat/hashcat.py

```diff
@@ -0,0 +1,104 @@
+import argparse
+import hashlib
+import itertools
+import string
+import time
+
+# Unterstützte Hash-Algorithmen
+hash_modes = {
+    '0': 'md5',
+    '1': 'sha1',
+    '2': 'sha256',
+    '3': 'sha512'
+}
+
+def hash_function(mode):
```

Comment:
rename function to something with a better name, e.g. `translate_mode_to_hash_algorithm`